config/configgrpc/configgrpc.go

```diff
@@ -175,10 +178,27 @@ func (gcs *GRPCClientSettings) ToDialOptions() ([]grpc.DialOption, error) {
 		})
 		opts = append(opts, keepAliveOption)
 	}
-
+	
+	if gcs.BalancerName != "" {
+		err = validateBalancerName(gcs.BalancerName)
+		if err != nil {
+			return nil, err
+		}
+		opts = append(opts, grpc.WithBalancerName(gcs.BalancerName))
+	}
 	return opts, nil
 }
 
+func validateBalancerName(balancerName string) (err error) {
```

Comment:
Can we explicitly check here for the right values. I know we will need work when new format is accepted but that way we control our backwards compatible story

Reply:
Sorry. I couldn't get what you mean by backwards compatible story? Do you mean the API change for setting the balancerName?

Reply:
So, I ended up removing the other call to `grpc.WithBalancerName` in `validateBalancerName` to  fix the lint errors.